blackjack.js

```diff
@@ -19,10 +43,43 @@ const player; // TODO
  * @returns {boolean} blackJackScore.isSoft
  */
 const calcPoints = (hand) => {
-  // CREATE FUNCTION HERE
+  // CREATE FUNCTION HERE 
+  total = 0;
+  let calcHand = [hand];
 
+  for(let h=0; h < calcHand.length; h++){
+    switch(true){
+      case ({displayVal} === 'Ace'):
+        total += 11
+        break;
+      case ({val} > 1 && {val} < 10):
+        total += {val}
+        break;
+      case ({displayVal} === '10'):
+        total += 10
+        break;
+      case ({displayVal} === 'Jack'):
+        total += 10
+        break;
+      case ({displayVal} === 'Queen'):
+        total += 10
+        break;
+      case ({displayVal} === 'King'):
+          total += 10
+          break
+    }
+    let isSoft = true;
+    if (calcHand /* has Ace as 11pts */) {
+      isSoft = true;
+    } else{ 
+      isSoft = false;
+      }
+    // { val: 11, displayVal: 'Ace', suit: 'diamonds' },
+  }
+  return total, isSoft;   
```

Comment:
almost, remember from the specification in the homework page: `This function should return an object` . . . so `return {total, isSoft}` should do the trick.